stream.go

```diff
@@ -323,6 +323,8 @@ func (s *stream) signalWrite() {
 	}
 }
 
+// SetDeadline sets the read deadline
```

Comment:
Nit: Please use the proper comment rules for godoc (i.e. fullstops at the end of each line / sentence, capitalize new lines / sentences) throughout this commit.